packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -538,9 +566,13 @@ export class BlobManager {
             );
             const blobId = pathParts[2];
 
-            // The unused blobId could be a localId. If so, remove it from the redirect table and continue. The
-            // corresponding storageId may still be used either directly or via other localIds.
-            if (this.redirectTable?.has(blobId)) {
+            // GC tombstones these blobs
+            if (tombstone) {
+                this.tombstonedBlobs.add(blobId);
+                continue;
+            } else if (this.redirectTable?.has(blobId)) {
+                // The unused blobId could be a localId. If so, remove it from the redirect table and continue. The
```

Comment:
This comment is outdated and needs to be updated. Maybe I will do that in a separate PR. A lot of things has changed since I wrote this code.